xiaomi_vac/plugin.yaml

```diff
@@ -29,15 +27,24 @@ logic_parameters: NONE
 
 # Definition of parameters to be configured in etc/plugin.yaml
 parameters:
-    ip: 
-        de: 'IP Adresse des Roboters '
-        en: 'IP Adress of the Vakuum'
-    read_cycle: 
-        de: 'Standart Zyklus 12'
-        en: 'default cycle time 12'
-    token: 
-        de: 'Token für die Kommunikation'
-        en: 'Token for Communication'  
+    ip:
+        type: ipv4
+        default: mandatory
+        description:
+            de: 'IP Adresse des Saug-Roboters '
+            en: 'IP Adress of the Vakuum Cleaner'
+    read_cycle:
+        type: int
+        default: 12
+        description:
+            de: 'Standart Zyklus zum Lesen der Paramter vom Gerät'
+            en: 'Default cycle time for reading'
+    token:
+        type: str
+        default: mandatory
```

Comment:
Soll hier wirklich der String 'mandatory' in den Parameter eingetragen werden, oder soll es eine Pflichtangabe ```mandatory: True``` sein?